aspnet-runtime-6.0/Dockerfile

```diff
@@ -2,8 +2,8 @@
 # SPDX-License-Identifier: MIT
 #!BuildTag: bci/dotnet-aspnet:6.0
 #!BuildTag: bci/dotnet-aspnet:6.0-%RELEASE%
-#!BuildTag: bci/dotnet-aspnet:6.0.20
-#!BuildTag: bci/dotnet-aspnet:6.0.20-%RELEASE%
+#!BuildTag: bci/dotnet-aspnet:1.x86_64.rpm
```

Comment:
This looks wrong

Reply:
Argh, indeed. I have a hunch where this is coming from…